server/neptune/workflows/internal/deploy/terraform/runner_test.go

```diff
@@ -89,9 +111,26 @@ func parentWorkflow(ctx workflow.Context, r request) (response, error) {
 func TestWorkflowRunner_Run(t *testing.T) {
 	ts := testsuite.WorkflowTestSuite{}
 	env := ts.NewTestWorkflowEnvironment()
-
 	env.RegisterWorkflow(testTerraformWorkflow)
 
+	a := &testDbActivities{}
+	env.RegisterActivity(a)
+
+	env.OnActivity(a.FetchLatestDeployment, mock.Anything, activities.FetchLatestDeploymentRequest{
+		FullRepositoryName: "owner/test",
+	}).Return(activities.FetchLatestDeploymentResponse{}, nil)
+
+	env.OnActivity(a.StoreLatestDeployment, mock.Anything, activities.StoreLatestDeploymentRequest{
+		DeploymentInfo: deployment.Info{
```

Comment:
It'd be nice if we could use the same `DeploymentInfo` object and pass into the activity. But since `DeploymentInfo` object is defined inside the terraform workflow package, it's not possible to reference that object in the activity since it creates a cyclic dependency. Ideally, we should be able to share this object both in the workflow code as well as the activity code. 

One way we could do that is have a following dir structure:
```
/internal
  /terraform
    // Contains objects shared by terraform workflow and activities 
  /deploy
    // Contains objects shared by deploy workflow and activities
  /workflows
    /deploy
        // Contains deploy workflow code
    /terraform
       // Contains terraform workflow code
```

This way we have one extra layer to separate out the workflow code and a shared dir for both the workflows to share objects with the activity code similar to how we do it for the root object. 

Or a simpler approach would be to just use the deployment.Info object everywhere instead of `terraform.DeploymentInfo`

Reply:
I'm not super into the proposed package structure, that's a bit confusing tbh. I don't think we should be sharing types between the two in general.